main/vstring.c

```diff
@@ -81,12 +81,12 @@ extern vString *vStringNew (void)
 
 extern void vStringPut (vString *const string, const int c)
```

Comment:
As far as I know `inline` function should be at `*.h file.
Could you move vStringPut to vstring.h, add inline keyword?
I'm not sure whether `static` should be added or not.